particles/Native/Wasm/source/TestParticle.kt

```diff
@@ -91,8 +91,8 @@ class TestParticle : Particle() {
 
         eventHandler("add") {
           val newData = data.get()!!
-          newData.num = newData.num + 2
-          newData.txt = newData.txt + "!!!!!!"
+          newData.num = newData.num?.let { it + 2 } ?: 0.0
+          newData.txt = (newData.txt ?: "") + "!!!!!!"
```

Comment:
Nit: could be `newData.txt = "${newData.txt ?: ""}!!!!!!"`

Reply:
Or, if it's fine to use `null` in the string, you can just make it `"${newData.txt}!!!!!!"` which would render as "null!!!!!!"